### PR TITLE
feat: use environment variable to check rules format

### PR DESCRIPTION
**Pull request by christyjacob4**
## What does this PR do?
Use an environment variable instead of the version to decide the mode for fetching and creating domains ( rules ) 

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Screenshots may also be helpful.)

## Related PRs and Issues

- (Related PR or issue)

## Checklist

- [ ] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [ ] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: app/controllers/general.php [hunk 1]
```diff
@@ -528,10 +528,10 @@ function router(App $utopia, Database $dbForConsole, callable $getProjectDB, Swo
                     $mainDomain = $envDomain;
                 } else {
                     // TODO: @christyjacob remove once we migrate the rules in 1.7.x
-                    if (version_compare(APP_VERSION_STABLE, '1.7.0', '<')) {
-                        $domainDocument = $dbForConsole->findOne('rules', [Query::orderAsc('$id')]);
-                    } else {
+                    if (System::getEnv('_APP_RULES_FORMAT', null) === 'md5') {
```
Comment (lohanidamodar) [inline app/controllers/general.php:531]: do we need `null` as default value here? it's already `null` by default right?

**Review by lohanidamodar — COMMENTED** [review]
Rest looks good. Let's make sure we search everywhere for `version_compare` to not miss anything

**Review by lohanidamodar — COMMENTED** [review]
Rest looks good. Let's make sure we search everywhere for `version_compare` to not miss anything

File: app/controllers/general.php (hunk 1 above)
Comment (christyjacob4) [reply app/controllers/general.php:531]: Got you. Yeap, will revert it

**Comment by christyjacob4** [comment]
@lohanidamodar checked, looks good 👍

**Review by lohanidamodar — APPROVED** [review]

**Pull request merged**